crates/acir/src/native_types/arithmetic.rs

```diff
@@ -34,13 +34,45 @@ impl Default for Arithmetic {
     }
 }
 
+impl std::fmt::Display for Arithmetic {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        if self.mul_terms.is_empty() && self.linear_combinations.len() == 1 && self.q_c.is_zero() {
+            write!(f, "x{}", self.linear_combinations[0].1.witness_index())
+        } else {
+            write!(f, "%{:?}%", crate::circuit::gate::Gate::Arithmetic(self.clone()))
+        }
+    }
+}
+
 impl Arithmetic {
     pub const fn can_defer_constraint(&self) -> bool {
         false
     }
     pub fn num_mul_terms(&self) -> usize {
         self.mul_terms.len()
     }
+
+    pub fn get_value(
```

Comment:
This method seems to be misplaced as in ACIR, we are not concerned about the actual value that the witnesses hold